plugins/modules/redfish_command.py

```diff
@@ -109,7 +109,8 @@
   timeout:
     description:
       - Timeout in seconds for HTTP requests to OOB controller.
-    default: 10
+      - The default value for this param is C(10) but that is being deprecated
+        and it will be replaced with C(60) in community.general 7.5.0.
```

Comment:
The next release is 7.5.0, and after that the next one will be 8.0.0. Let's only change this in 9.0.0 so users have a chance to see the deprecation message.
```suggestion
        and it will be replaced with C(60) in community.general 9.0.0.
```